.github/workflows/check-benchmarks.yml

```diff
@@ -59,9 +62,16 @@ jobs:
           rustup override unset
           rustup show
           rustup target add wasm32-unknown-unknown
+      - name: Install frame-omni-bencher
+        shell: bash
+        run: |
+          curl -sL https://github.com/paritytech/polkadot-sdk/releases/download/$FRAME_OMNI_BENCHER_RELEASE_VERSION/frame-omni-bencher -o frame-omni-bencher
+          chmod +x ./frame-omni-bencher
+          ./frame-omni-bencher --version
       - name: Run benchmarks
         shell: bash
         run: |
+          ln -s $(pwd)/frame-omni-bencher ./scripts/frame-omni-bencher
```

Comment:
This line is not necessary
```suggestion
```

Reply:
Are you sure, because in the workflow execution, we see that it can't find the script.
https://github.com/moonbeam-foundation/moonbeam/actions/runs/13726712455/job/38394779336

Reply:
Oh yes you are correct. Will remove.